### PR TITLE
Fix E2E tests execution

### PR DESCRIPTION
**Pull request by donpui**
It seems that after adding duplicity run check for CI, it ignored executing CI process for pull_request, where running E2E tests supposed to run.

Case visible here:
https://github.com/LeastAuthority/winden/actions/runs/3956222108/jobs/6775222052

I have enabled always run E2E that we would not get in such situation, when some steps are skipped.

FYI. @btlogy

### CODE REVIEW

File: client-e2e/test/specs/pages.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@ describe("Check internal Pages", () => {
         "Send files in real-time"
       );
 
-      await $('a[href="/about"').click();
+      await $('a[to="/about"').click();
```
Comment (JustusFT) [inline client-e2e/test/specs/pages.ts:11]: Seems there was a mistake in the `Link` component. Links should render `<a href="">` and not `<a to="">`. I fixed it here: https://github.com/LeastAuthority/winden/pull/133

Once it gets merged, we can revert this change to use `a[href=""]` again

**Comment by donpui** [comment]
@JustusFT take a look. Not sure why it fails in CI. Local run in docker container it don't get these issues. Also manual testing Edge, work ok in the same scenarious.

**Comment by JustusFT** [comment]
The check is already removed from the main branch so I'll close this.

For the tests failing in CI but passing locally: It seems that the `client-ci` service from https://github.com/LeastAuthority/winden/pull/50 does not apply the latest changes from the PR, so any test that fails will always fail. So I removed `client-ci` in favor of using `client`. The fix is in this commit: https://github.com/LeastAuthority/winden/pull/136/commits/05e07934a38e04a5a05df21679c709bd6a9ac6a9
